src/state/SceneScrollState.tsx

```diff
@@ -113,3 +113,37 @@ export const useSceneScrollHandler = (): SceneScrollHandler => {
 
   return handler
 }
+
+export const useSceneScrollWorkletHandler = () => {
```

Comment:
Please add a comment to this to explain why this is different than the one above.